ingress-nginx-controller-1.12.yaml

```diff
@@ -498,6 +502,71 @@ pipeline:
         && setcap -v cap_net_bind_service=+ep ${{targets.destdir}}/usr/bin/nginx
 
 subpackages:
+  - name: ingress-nginx-controller-bitnami-compat-${{vars.nginx-ingress-major-minor}}
+    description: Compatibility package with bitnami/nginx-ingress-controller image
+    dependencies:
+      provides:
+        - ingress-nginx-controller-bitnami-compat=${{package.full-version}}
+      runtime:
+        - bash # https://github.com/bitnami/charts/blob/db53e622ecf1be5a78d1ce683cec8baa41110fa3/bitnami/nginx-ingress-controller/templates/controller-deployment.yaml#L89
+        - coreutils
+        - curl
+        - gmp
+        - gnutls
+        - grep
+        - ingress-nginx-controller-compat-${{vars.nginx-ingress-major-minor}}
+        - libffi
+        #- libhogweed6 missing pkg/so # https://github.com/bitnami/containers/blob/942861e0cea74a6a07bceafb299180a2733c0bfe/bitnami/nginx-ingress-controller/1/debian-12/Dockerfile#L29
+        - libmaxminddb-dev
+        - librtmp
+        - libssh
+        - libtasn1
+        - nettle
+        - p11-kit
+        - procps
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: nginx-ingress-controller
+          version-path: ${{vars.nginx-ingress-major}}/debian-12
+      - name: Ensure symlinks
```

Comment:
Can we also have the following symlinks? https://github.com/bitnami/containers/blob/86622352cc33c05d23f6e97722c1364ca38ceeeb/bitnami/nginx-ingress-controller/1/debian-12/Dockerfile#L63-L64

Our most bitnami packages has this already.

Reply:
Thanks @Dentrax, the compat one this package depends on already provides them.